L1Trigger/VertexFinder/python/VertexProducer_cff.py

```diff
@@ -53,13 +53,13 @@
         # Minimum pt of tracks used to create vertex [GeV]
         VxMinTrackPt = cms.double(2.0),
         # Maximum pt of tracks used to create vertex [GeV]
-        VxMaxTrackPt = cms.double(50.0),
+        VxMaxTrackPt = cms.double(127.0),
         # When the track pt > VxMaxTrackPt, how should the tracks be considered
         #   -1 = tracks are valid
         #   0 = tracks are mismeasured and ignored/truncated
         #   1 = tracks are mismeasured and saturate at VxMaxTrackPt
         # Option '0' was used for the TDR, but '1' is used for the firmware
-        VxMaxTrackPtBehavior = cms.int32(0),
+        VxMaxTrackPtBehavior = cms.int32(1),
```

Comment:
Also took the opportunity to update the default values to match the FW and fix a bug reported [here](https://indico.cern.ch/event/1139008/#10-tracks-vertex)